web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -465,7 +479,43 @@ class AppConfig extends Component<Props, State> {
 
         const data = await response.json();
         const oldGroups = this.state.configGroups;
-        const newGroups = data.configGroups;
+        const validationErrors: ConfigGroupItemValidationErrors[] =
+          data.validationErrors;
+
+        // track errors at the form level
+        this.setState({ showValidationError: false });
+        let hasValidationError = false;
```

Comment:
We use this to set the error by the save config button and disable it. Since `setState` is async we need the local var `hasValidationError` to prevent multiple `setStates` from being called